### PR TITLE
Make _to_multi_line_cmd() public #259

### PR DESCRIPTION
**Pull request by christyanthony**
Task #259 

- Created a script with name "rename_to_multi_line_cmd.sh" 
- Did grep check for the newly replaced text and observed that the texts has been replaced.
- Ran linter to check for syntax error. No notable errors.

### CODE REVIEW

**Comment by christyanthony** [comment]
Task #259 

- Created a script with name "rename_to_multi_line_cmd.sh" 
- Did grep check for the newly replaced text and observed that the texts has been replaced.
- Ran linter to check for syntax error. No notable errors.

**Comment by samarth9008** [comment]
@christyanthony 

Go through this doc.  https://github.com/sorrentum/sorrentum/blob/master/docs/First_review_process.md

Usually, we do not commit those logs files and txt files or any temporary files or OS generated files. So pls remove them from your PR. 

Also name your PR according to the Issue.

While running the script, even your script would replace those text, so please modify the script after running it.

**Comment by christyanthony** [comment]
You mean I have to include only the newly created script and the folders where the function name changed and am allowed to remove other log and txt files? ?

**Comment by samarth9008** [comment]
> You mean I have to include only the newly created script and the folders where the function name changed and am allowed to remove other log and txt files? ?

Yes.

**Comment by christyanthony** [comment]
@samarth9008 , 
- I just removed the mentioned files
- named PR according to the issue
- Also, I checked the script after running it, I didn't see any change in there. Therefore, didn't make any changes.

File: helpers/test/outcomes/TestCheckDataFrame1.test_check_df_not_equal3/output/test_df.txt
Comment (DanilYachmenev) [inline helpers/test/outcomes/TestCheckDataFrame1.test_check_df_not_equal3/output/test_df.txt:1]: what about this and many other changed `.txt` files in this PR? these changes seem unnecessary and not related to this issue

File: helpers/test/outcomes/Test_purify_from_env_vars.test_end_to_end/tmp.final.actual.txt
Comment (DanilYachmenev) [inline helpers/test/outcomes/Test_purify_from_env_vars.test_end_to_end/tmp.final.actual.txt:1]: never commit `tmp` files, they are only for local use

File: im_v2/im_lib_tasks.py
Comment (DanilYachmenev) [inline im_v2/im_lib_tasks.py:1]: well, removing lib files is also not part of this PR
so pls revert

**Comment by DanilYachmenev** [comment]
> Go through this doc. https://github.com/sorrentum/sorrentum/blob/master/docs/First_review_process.md

@christyanthony pls do and treat your PR page accordingly
particularly I'm talking about labels

File: helpers/lib_tasks_utils.py [hunk 1]
```diff
@@ -155,7 +155,7 @@ def _to_single_line_cmd(cmd: Union[str, List[str]]) -> str:
 
 
 # TODO(Grisha): make it public #755.
```
[the remark this answers is not in the document]
Comment (christyanthony) [reply helpers/lib_tasks_utils.py:157]: You mean to remove it and do a commit again?

File: helpers/test/outcomes/TestCheckDataFrame1.test_check_df_not_equal3/output/test_df.txt
Comment (christyanthony) [reply helpers/test/outcomes/TestCheckDataFrame1.test_check_df_not_equal3/output/test_df.txt:1]: Yesterday when I did the pull request, some of the log and .txt files has also been pushed and Samarth asked me remove them and did the same as per the request.

File: helpers/test/outcomes/Test_purify_from_env_vars.test_end_to_end/tmp.final.actual.txt
Comment (christyanthony) [reply helpers/test/outcomes/Test_purify_from_env_vars.test_end_to_end/tmp.final.actual.txt:1]: Will remove this files too

**Review by samarth9008 — COMMENTED** [review]
LG.

File: helpers/test/outcomes/TestCheckDataFrame1.test_check_df_not_equal3/output/test_df.txt
Comment (DanilYachmenev) [inline helpers/test/outcomes/TestCheckDataFrame1.test_check_df_not_equal3/output/test_df.txt:1]: pls uncommit this file and all the such files
in fact your PR should do changes only related to the task itself

go over `Files changed` section to check yourself
Снимок экрана 2023-06-22 в 00 44 31

basically, none of the test outcomes shoul've changed

**Review by DanilYachmenev — COMMENTED** [review]
overall, looks good
pls uncommit all the non-related files (test outcomes in this case) and ready to merge

File: helpers/test/outcomes/TestCheckDataFrame1.test_check_df_not_equal3/output/test_df.txt
Comment (christyanthony) [reply helpers/test/outcomes/TestCheckDataFrame1.test_check_df_not_equal3/output/test_df.txt:1]: Will work on this

Comment (christyanthony) [reply helpers/test/outcomes/TestCheckDataFrame1.test_check_df_not_equal3/output/test_df.txt:1]: I am confused about what I understood. Please let me know if my understanding was correct and also correct if there's anything wrong.
- Firstly, all test or log related files shouldn't be there in the PR. Therefore, I deleted them as per request.
- Now these deleted files shouldn't be there in the PR and therefore asking me to uncommit the file. (Eventually, the goal is to only include the task related changes in the PR).
- when I googled about this, it was said to do git checkout HEAD^ -- filepath
If I do this it will restore the file and brings it back. But all my commits have the files either being deleted or being available in the previous commits.
-Then I thought of another approach where to reset the first commit(because this is where these files has been added in the first place) and unstage these files and commit again. But I couldn't see these test files after doing reset in the staged file list. 

I am bit confused. Could you please suggest a solution or any possible documentation

Comment (christyanthony) [reply helpers/test/outcomes/TestCheckDataFrame1.test_check_df_not_equal3/output/test_df.txt:1]: @DanilYachmenev , could you please respond on this

Comment (samarth9008) [reply helpers/test/outcomes/TestCheckDataFrame1.test_check_df_not_equal3/output/test_df.txt:1]: Maybe you can try using `git revert` command.

Comment (christyanthony) [reply helpers/test/outcomes/TestCheckDataFrame1.test_check_df_not_equal3/output/test_df.txt:1]: I actually used a different method.

**Comment by samarth9008** [comment]
You have to remove `PR_for_authors` label once you add the `reviewers` label so its not confusing for us.

**Comment by samarth9008** [comment]
LGTM!

**Comment by DanilYachmenev** [comment]
> I will first try doing it with a different branch. So here you are expecting only the newly created script but not any folders where the function has been replaced. For example, replacing "_to_multi_line_cmd" with "to_multi_line_cmd" in helpers/lib_tasks_utils.py, this change shouldn't be involved in the new branch's commit. @DanilYachmenev Could you please confirm?

yes + remove TODO above the function itself
then we merge your PR with these changes

this will allow me to simply run your script on `cmamp` repo so we can test if everything is correct
